src/test/java/uk/gov/hmcts/reform/jobscheduler/services/jobs/TriggerConverterTest.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.jobscheduler.services.jobs;
+
+import org.junit.Test;
+import org.quartz.SimpleScheduleBuilder;
+import uk.gov.hmcts.reform.jobscheduler.model.Trigger;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.jobscheduler.SampleData.validJobEmptyInterval;
+
+public class TriggerConverterTest {
+
+    @Test
+    public void should_use_default_scheduler_builder_and_convert_triggers_correctly() {
+        Trigger originalTrigger = validJobEmptyInterval().trigger;
+        org.quartz.Trigger trigger = TriggerConverter.toQuartzTrigger(originalTrigger);
+
+        // dummy assertion to remind in the future of different schedulers in use and verify
+        assertThat(trigger.getScheduleBuilder()).isOfAnyClassIn(SimpleScheduleBuilder.class);
```

Comment:
As the converter creates the trigger, we should also have an assertion for the trigger's start date (and maybe the type)

Reply:
all fields are tested recursively in the line 23.

Reply:
I think we should test each type of conversion separately.

Testing of `toQuartzTrigger` should verify if the properties of the quartz trigger match the properties of the original platform trigger.
Testing of `toPlatformTrigger` could use the (already tested) `toQuartzTrigger` to create a Quartz trigger, convert it back to platform version and compare to the original.

Currently, we're only testing the second thing, i.e. `toPlatform(toQuartz(A)) is same as A`, but not `toQuartz(A) is as expected`.

Reply:
sounds like you want to test library's models as well as our models